.github/workflows/owasp-scanner.yml

```diff
@@ -0,0 +1,52 @@
+name: OWASP Scanner
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Artifact version'
+        required: true
+        type: string
+      component:
+        description: 'Artifact component'
+        required: false
+        type: string
+        default: 'liquibase'
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Artifact version'
+        required: true
+        type: string
+      component:
+        description: 'Artifact component'
+        required: true
+        type: string
+        default: 'liquibase'
+
+
+jobs:
+  scan:
+    runs-on: ubuntu-22.04
+    steps:
+
+     - name: Download OWASP Dependency-Check
+       run: |
+        wget https://github.com/jeremylong/DependencyCheck/releases/download/v9.2.0/dependency-check-9.2.0-release.zip
```

Comment:
I see dependencycheck has [a maven plugin](https://github.com/jeremylong/DependencyCheck?tab=readme-ov-file#maven-plugin) is there a reason we're downloading the release zip instead of using that?

Reply:
I think it's because @jandroav automated the manual process that we did before, and it was done using this external tool. But @abrackx got a really good point! Should be add a new profile and just run it?
I think it would be easier to track owasp versions this ways as dependabot would maintain it for us.

Reply:
Yeah, good point. but where to execute it?

Reply:
also keep in mind that this workflow can work with every other future component like extensions without adding maven plugins to anywhere

Reply:
Makes sense for me.